src/main/resources/wa-task-initiation-ia-asylum.dmn

```diff
@@ -14,7 +14,10 @@
       </input>
       <input id="InputClause_0a0i7vo" label="Appeal Type" camunda:inputVariable="appealType">
         <inputExpression id="LiteralExpression_18qwedg" typeRef="string">
-          <text></text>
+          <text>if(caseData != null and caseData.Data != null and caseData.Data.appealType != null) then
+caseData.Data.appealType
```

Comment:
is this correct? 
should it be 
```
caseData.data.appealType
```

Reply:
In AdditionalData , the data field used as Data json property.
`@ToString
@EqualsAndHashCode
@Builder
public class AdditionalData {

    private final Map<String, Object> data;
    private final Map<String, JsonNode> definition;

    @JsonCreator
    public AdditionalData(@JsonProperty("Data") Map<String, Object> data,
                          @JsonProperty("Definition") Map<String, JsonNode> definition) {
        this.data = data;
        this.definition = definition;
    }

    public Map<String, Object> getData() {
        return data;
    }

    public Map<String, JsonNode> getDefinition() {
        return definition;
    }
}`

Reply:
let's take this offline